tests/test_plans/test_plan_rel_0.2.md

```diff
@@ -0,0 +1,148 @@
+|---|----------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------|---------|--------------------------------------------------------------------------|
+| 1 | Webhook implementation                                   | Tests will be added later, tested using the Bulk demo                                                                                                  |                                                      | PASSED  |                                                                          |
+| 2 | Add time range filter for bulk API                       | Tests will be added later, tested using the Bulk demo                                                                                                  |                                                      | PASSED  |                                                                          |
+| 3 | Bulk API error handling with New JSON                    | Tests will be added later, tested using the Bulk demo                                                                                                  |                                                      | PASSED  |
+| 4 | Database updates for authentication                      | [New tests added](https://github.com/kruize/autotune/blob/mvp_demo/tests/scripts/local_monitoring_tests/Local_monitoring_tests.md#authentication-test) | [1307](https://github.com/kruize/autotune/pull/1307) | PASSED  |                                                                          |
+| 5 | Datasource Exception handler                             | Regression testing                                                                                                                                     |                                                      |         | Issue seen [1395](https://github.com/kruize/autotune/issues/1395)        |
+| 6 | Bulk test cases                                          | [New tests added](https://github.com/kruize/autotune/blob/master/tests/scripts/local_monitoring_tests/Local_monitoring_tests.md#bulk-api-tests)        | [1372](https://github.com/kruize/autotune/pull/1372) | PASSED  |                                                                          |
+| 7 | Update kruize default cpu/memory resources for openshift | Tested with existing remote monitoring tests                                                                                                           |                                                      | FAILED  | DB Shutdown issue [1393](https://github.com/kruize/autotune/issues/1393) |
```

Comment:
This can be marked as PASSED as it worked fine after fixing the resources for the tests

Reply:
Changed the result to PASSED